src/utils/available-rpc.ts

```diff
@@ -1,5 +1,5 @@
 /// <reference types="../types/global" />
-import { JsonRpcProvider, Network } from 'ethers';
+import { JsonRpcProvider, Network, Provider, WebSocketProvider } from 'ethers';
```

Comment:
`Provider` here should import just the type?

Reply:
Good catch